privacyidea/static/components/login/views/enter-response.html

```diff
@@ -4,8 +4,8 @@
             <p>{{ challenge_message }}</p>
         </div>
         <div class="center"
-            ng-show="image">
-            <img width="250" src="{{ image }}"/>
+            ng-if="image">
```

Comment:
Why change this to ng-if?

Reply:
``ng-show`` has the following disadvantage: If ``image`` is false, the ``<img>`` below actually gets added to the page (but is subsequently hidden). And this results in a request for ``/false``, because of ``src="{{ image }}"`` :-)

Reply:
Ah, ok. This is false. And we do not change the image later, so this is ok...